package.json

```diff
@@ -14,12 +15,10 @@
   "author": {
     "name": "chenglou"
   },
-  "peerDependencies": {
-    "ocaml": " >= 4.2.0  < 4.7.0"
-  },
   "devDependencies": {
     "@esy-ocaml/merlin": "*",
-    "ocaml": "~4.6.0"
+    "ocaml": "~4.6.0",
+    "@esy-ocaml/reason": "3.3.3"
```

Comment:
Curious why you put reason as a devDependency? With esy everything which needs to be available during building the package should go into dependencies (soon into buildDependencies).

Reply:
Good question. So Reason is/was already specified as a dependency with a much looser version requirement ("^3.0.0.0"), which is desirable for consumers of our package. 

One of our tests was failing however because across versions of reason, an error code had changed (a 1494 syntax error became a 1499 syntax error or vice versa), which caused a test to fail that had been essentially snapshotting on that output. By locking down the version of reason in devDependencies we ensuring that our tests are resilient to this kind of thing.